src/ServicePulse.Host/app/js/views/pending_retries/controller.js

```diff
@@ -0,0 +1,375 @@
+﻿/// <reference path="service.js" />
+;
+(function(window, angular, undefined) {
+    "use strict";
+
+    function controller(
+        $scope,
+        $timeout,
+        $location,
+        $moment,
+        $filter,
+        scConfig,
+        toastService,
+        sharedDataService,
+        notifyService,
+        serviceControlService,
+        endpointsService, 
+        redirectService,
+        pendingRetryService) {
+
+        var vm = this;
+
+        var notifier = notifyService();
+        
+        vm.sortButtonText = '';
+        vm.sortDirection = 'asc';
+        vm.allMessagesLoaded = false;
+        vm.loadingData = false;
+
+        vm.timeGroup = {
+            amount: undefined,
+            unit: undefined,
+            buttonText: 'All Pending Retries',
+            selected: function () {
```

Comment:
`selected` doesn't sound like an appropriate name. It implies a yes/no flag to me, but this returns a duration.